src/components/Login.tsx

```diff
@@ -66,6 +86,9 @@ const Login = ({ onSubmit, title }: LoginProps) => {
           isLoading={isSubmitting}
           type="submit"
           borderColor="black"
+          onClick={() => {
+            inputs = getValues(['username', 'password']);
```

Comment:
This `onClick` handler won't work as intended, you're mutating the `inputs` variable but React has no way for keeping track of this variable (there is also no guarantee that this `onClick` fires before the `onSubmit` handler fires. Not for this task, but if you really need to mutate a variable in React, and not do things in state, use the `useRef` hook.

For this task, the `onClick` handler here is also redundant. According to the docs for react-hook-form, the form data is passed as the first argument to the submit handler so you don't need to get values this way